docs/examples/SPA.md

```diff
@@ -0,0 +1,66 @@
+## Using offline-plugin in a Single Page Application (SPA)
+
+In a SPA, only the inital HTML is served from the server. Any routing beyond that takes place on the client. So how should we configure offline-plugin to handle this?
+
+The first thing we should do is to tell offline-plugin to cache the root `/` route. We can do this by adding it to `caches.main`.
+
+```javascript
+caches: {
+  main: [
+    '/'
```

Comment:
Simply add it to `externals`, then it's automatically being used in caches, so not need of the next section. Also should be a mention of `html-webpack-plugin`, i.e. if they use it to generate `index.html` then no need to add that.